### DISCUSSION TITLE
Socket integration [May 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2066] [Discord Threads]
> 🙋‍♂️ **b6temd**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@307205291609423874> Could you please expand on your requirement, use-case and other relevant details?  
#  
> 💁 **o1lab**  
#### It's part of feature requests  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/970922363300507648)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
